internal/services/kusto/identity.go

```diff
@@ -124,9 +124,10 @@ func expandTrustedExternalTenants(input []interface{}) *[]kusto.TrustedExternalT
 	output := make([]kusto.TrustedExternalTenant, 0)
 
 	for _, v := range input {
-		// TODO 3.0: remove MyTenantOnly
-		if v.(string) == "MyTenantOnly" {
-			return &[]kusto.TrustedExternalTenant{}
+		if !features.ThreePointOhBetaResources() {
```

Comment:
```suggestion
		if !features.ThreePointOhBeta() {
```

Reply:
although I have a feeling I may have already made this change, which'd be the conflict?